crates/svm-runtime-c-api/src/macros.rs

```diff
@@ -72,6 +72,7 @@ macro_rules! impl_into_svm_byte_array {
                 $crate::svm_byte_array {
                     bytes: bytes.as_ptr(),
                     length,
+                    capacity: length,
```

Comment:
at line `64` above we can extract the real capacity:

```rust
let capacity = bytes.capacity() as u32;
```

Reply:
`bytes` is an array (hence without `capacity()`), no?

Reply:
I don't remember, if it's then indeed `capacity = length`